### PR TITLE
Gangams/enable release pipe line

### PR DESCRIPTION
**Pull request by ganga1980**
This PR has the following changes 

1.  Enable release pipeline to automatically deploy bits on to CIDEV AKS cluster in Build subscription
2.  Instructions for the release pipeline 
3. Readme updates
4. updates to gitignore and gitattributes files
5. Instructions for production release
6. Fix line endings to LF for yaml so that sed command can be used.

### CODE REVIEW

File: kubernetes/omsagent.yaml [hunk 1]
```diff
@@ -1,728 +1,731 @@
...
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 1
+              memory: 750Mi
+            requests:
+              cpu: 150m
+              memory: 250Mi
+          env:
+            # azure devops pipeline uses AKS_RESOURCE_ID and AKS_REGION hence ensure to uncomment these
```
Comment (vishiy) [inline kubernetes/omsagent.yaml:483]: why change default from non-aks to AKS ?

File: kubernetes/container-azm-ms-agentconfig.yaml [hunk 2]
```diff
@@ -1,105 +1,105 @@
-kind: ConfigMap
```
Comment (rashmichandrashekar) [inline kubernetes/container-azm-ms-agentconfig.yaml:1]: if its whitespace only, can you undo changes to this file?

File: kubernetes/omsagent.yaml (hunk 1 above)
Comment (ganga1980) [reply kubernetes/omsagent.yaml:483]: > why change default from non-aks to AKS ?

> pipelines are replacing these place-holders to deploy onto ci_dev and ci_prod clusters. do you see any issue  changing this?

File: kubernetes/container-azm-ms-agentconfig.yaml (hunk 2 above)
Comment (ganga1980) [reply kubernetes/container-azm-ms-agentconfig.yaml:1]: > if its whitespace only, can you undo changes to this file?

> sure, let me do it.

File: kubernetes/omsagent.yaml (hunk 1 above)
Comment (ganga1980) [reply kubernetes/omsagent.yaml:483]: > > why change default from non-aks to AKS ?
> 
> > pipelines are replacing these place-holders to deploy onto ci_dev and ci_prod clusters. do you see any issue  changing this? for non-AKS, clusters, release pipelines will uses the HELM chart which I am planning to do next PR.

File: kubernetes/container-azm-ms-agentconfig.yaml (hunk 2 above)
Comment (ganga1980) [reply kubernetes/container-azm-ms-agentconfig.yaml]: > > if its whitespace only, can you undo changes to this file?
> 
> > sure, let me do it.

Done.

File: kubernetes/omsagent.yaml (hunk 1 above)
Comment (vishiy) [reply kubernetes/omsagent.yaml:483]: no, not an issue. should be ok.

**Pull request merged**